nana/modules/lookup.py

```diff
@@ -0,0 +1,67 @@
+from nana.helpers.aiohttp_helper import AioHttp
+from datetime import datetime
+from asyncio import sleep
+
+from pyrogram import filters
+from pyrogram.types import InlineKeyboardButton, InlineQueryResultArticle, InlineQueryResultArticle, InputTextMessageContent, InlineKeyboardMarkup
+from pyrogram.errors import PeerIdInvalid
+
+from nana import app, Command, AdminSettings, setbot, edrep
+
+@app.on_message(filters.user(AdminSettings) & filters.command("lookup", Command))
+async def lookup(client, message):
+    cmd = message.command
+    if not message.reply_to_message and len(cmd) == 1:
+        get_user = message.from_user.id
+    elif len(cmd) == 1:
+        if message.reply_to_message.forward_from:
+            get_user = message.reply_to_message.forward_from.id
+        else:
+            get_user = message.reply_to_message.from_user.id
+    elif len(cmd) > 1:
+        get_user = cmd[1]
+        try:
+            get_user = int(cmd[1])
+        except ValueError:
+            pass
+    try:
+        user = await client.get_chat(get_user)
+    except PeerIdInvalid:
+        await edrep(message, text="I don't know that User.")
+        sleep(2)
+        await message.delete()
+        return
+    url = f'https://api.intellivoid.net/spamprotection/v1/lookup?query={user.id}'
+    a = await AioHttp().get_json(url)
+    response = a['success']
+    if response == True:
+        date = a["results"]["last_updated"]
+        stats = f'**◢ Intellivoid• SpamProtection Info**:\n'
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)